packages/webidl/webidl.1.2/opam

```diff
@@ -11,7 +11,7 @@ depends: [
   "ocaml" {>= "4.03.0"}
   "ocamlfind" {build & >= "1.7.1"}
   "ocamlbuild" {build & >= "0.9.3"}
-  "menhir" {build & >= "20170607" & < "20211215"}
+  "menhir" {build & >= "20170607"}
```

Comment:
`webidl.1.2` does not use `menhir` at all.
```suggestion
```

Reply:
Thanks